CHANGES

```diff
@@ -1,3 +1,55 @@
+Changes from 8.6 to 8.6.1
+=========================
+
+- Fix #5380: Default colors for CoqIDE are actually applied.
+- Fix plugin warnings
+- Use more neutral wording instead of mentioning CoqIDE
```

Comment:
Some of the messages mentioned CoqIDE specifically, but those messages also apply to PG/xml or vsCoq. See PR #812.